coldfront/core/user/views.py

```diff
@@ -41,13 +44,36 @@
 from coldfront.core.utils.common import (import_from_settings,
                                          utc_now_offset_aware)
 from coldfront.core.utils.mail import send_email_template
-
 logger = logging.getLogger(__name__)
 EMAIL_ENABLED = import_from_settings('EMAIL_ENABLED', False)
 if EMAIL_ENABLED:
     EMAIL_TICKET_SYSTEM_ADDRESS = import_from_settings(
-        'EMAIL_TICKET_SYSTEM_ADDRESS')
+        'EMAIL_TICKET_SYSTEM_ADDRESS') 
+
+#for staff and super users  
+class SendEmailView(LoginRequiredMixin, UserPassesTestMixin, FormView):
+    template_name = 'user/send_email_form.html'
+    form_class = SendEmailForm
+    success_url = 'send-email'
+    context = dir()
```

Comment:
Hm, not sure what `dir()` is doing here (there might be a good reason)--can you explain?